test/unittests/leb128_test.cpp

```diff
@@ -212,32 +209,31 @@ TEST(leb128, decode_out_of_buffer)
 
 TEST(leb128, decode_s64)
 {
-    // clang-format off
-    std::vector<std::pair<bytes, int64_t>> testcases = {
-        {{0}, 0}, // 0
-        {{0x80, 0x80, 0x00}, 0}, // 0 with leading zeroes
-        {{1}, 1},
-        {{0x81, 0x80, 0x80, 0x00}, 1}, // 1 with leading zeroes
-        {{0x81, 0x80, 0x80, 0x80, 0x80, 0x80, 0x80, 0x80, 0x80, 0x00}, 1}, // 1 with max leading zeroes
-        {{0x7f}, -1},
-        {{0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0x7f}, -1}, // -1 with leading 1s
-        {{0x7e}, -2},
-        {{0xfe, 0x7f}, -2}, // -2 with leading 1s
-        {{0xfe, 0xff, 0x7f}, -2},  // -2 with leading 1s
-        {{0xe5, 0x8e, 0x26}, 624485},
-        {{0xe5, 0x8e, 0xa6, 0x80, 0x80, 0x00}, 624485}, // 624485 with leading zeroes
-        {{0xc0, 0xbb, 0x78}, -123456},
-        {{0x9b, 0xf1, 0x59}, -624485},
-        {{0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0x00}, 562949953421311}, // bigger than int32
-        {{0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0x80, 0x80, 0x00}, 562949953421311}, // bigger than int32 with zeroes
+    std::pair<std::string_view, int64_t> test_cases[]{
```

Comment:
Why not constexpr here and below?
```suggestion
    constexpr std::pair<std::string_view, int64_t> test_cases[]{
```